addons/event/models/event_event.py

```diff
@@ -102,9 +102,12 @@ def _get_default_stage_id(self):
         event_stages = self.env['event.stage'].search([])
         return event_stages[0] if event_stages else False
 
+    def _default_description(self):
+        return self.env['ir.ui.view']._render_template('event.event_default_descripton')
```

Comment:
@JKE-be I would have written `self.env.ref('event.event_default_descripton').render_template()` or something like that but not sure? What about `viewref` when not in website?

Reply:
both are valid imo

self.env['ir.ui.view']._render_template('event.event_default_descripton')
==
self.env.ref('event.event_default_descripton')._render()

In all case, render became recently private and view are no mode readable for public/portal user. 

KR